.misc/deploy.pypi.sh

```diff
@@ -6,22 +6,28 @@ set -e
 # Get environment variables, readily decrypted by rultor
 source ../rultor_secrets.sh
 
-# Ship it!
-echo "Uploading coala to pypi"
-pip3 install twine wheel
-python3 setup.py sdist bdist_wheel
-# Upload one by one to avoid timeout
-twine upload dist/* -u "$PYPIUSER" -p "$PYPIPW" 2>&1 | tee twine_output.txt
-if [ "${PIPESTATUS[0]}" -ne 0 ]; then
-    SEARCH_STR="500 Server Error"
-    if grep -q "$SEARCH_STR" twine_output.txt; then
-        echo "Server error 500"
-        exit 1
+# Make sure the user is a maintainer
+if curl "https://api.github.com/teams/1238757/memberships/${author}?access_token=${GITHUB_TOKEN}" | grep -q "active"; then
+    # Ship it!
+    echo "Uploading coala to pypi"
+    pip3 install twine wheel
+    python3 setup.py sdist bdist_wheel
+    # Upload one by one to avoid timeout
+    twine upload dist/* -u "$PYPIUSER" -p "$PYPIPW" 2>&1 | tee twine_output.txt
+    if [ "${PIPESTATUS[0]}" -ne 0 ]; then
+        SEARCH_STR="500 Server Error"
+        if grep -q "$SEARCH_STR" twine_output.txt; then
+            echo "Server error 500"
+            exit 1
+        fi
     fi
-fi
-rm twine_output.txt
+    rm twine_output.txt
 
-echo "Installing coala from pypi"
-pip3 install --pre coala==`cat coalib/VERSION` --upgrade
-echo coala versions: pip=`coala -v` repo=`cat coalib/VERSION`
-[ `coala -v` = `cat coalib/VERSION` ]
+    echo "Installing coala from pypi"
+    pip3 install --pre coala==`cat coalib/VERSION` --upgrade
+    echo coala versions: pip=`coala -v` repo=`cat coalib/VERSION`
+    [ `coala -v` = `cat coalib/VERSION` ]
+else
+    echo "You are not in the maintainers group."
+    exit 1
```

Comment:
rather than moving everything into an `if`, you can use the `if` to exit early, and the rest of the code can remain where it was.